tsconfig-build-options.json

```diff
@@ -6,7 +6,9 @@
         "declarationMap": true
     },
     "exclude": [
+        // Implies there's a types-index.d.ts already which shouldn't be overwritten
+        "**/types-index.js",
```

Comment:
I was first surprised this is necessary, but I guess when I used to manually author `.d.ts` files, I either was not emitting from `.js`, or I kept any `.js` out of reach of tsc. Thinking about it, there is no way for tsc to know that the `.d.ts` file was manually authored and not previously generated.